docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+        }
+      }
+    },
+
+    "tas-volunteer": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-volunteer",
+      "title": "tas-volunteer",
+      "description": "A volunteer item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["volunteer"]
+        },
+        "maxNumber": {
+          "description": "Max number of records allowed",
+          "type": "number"
+        }
+      }
+    },
+
+    "tas-skill": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "title": "tas-skill",
+      "description": "A skills item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["skill"]
+        },
+        "maxNumber": {
+          "description": "Max number of skill allowed",
+          "type": "number"
+        }
+      }
+    },
+
+    "tas-education": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-education",
+      "title": "tas-education",
+      "description": "A education item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["education"]
+        },
+        "maxNumber": {
+          "description": "Max number of education objects allowed",
+          "type": "number"
+        },
+        "educationTypes": {
+          "description": "Types of education",
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+
+    "tas-date": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-date",
+      "title": "tas-date",
+      "description": "A date item YYYY-MM-DD",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["date"]
+        },
+        "oldestDateAllowed": {
+          "description": "Oldest date that can be selected",
+          "type": "string"
+        },
+        "newestDateAllowed": {
+          "description": "Newest date that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-time": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-time",
+      "title": "tas-time",
+      "description": "A time item HH:MM:SS-HH:MM",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["time"]
+        },
+        "oldestTimeAllowed": {
+          "description": "Oldest time that can be selected",
+          "type": "string"
+        },
+        "newestTimeAllowed": {
+          "description": "Newest time that can be selected",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-datetime": {
```

Comment:
Copy and paste alert. Probably these three should be merged into a single date-time type with some additional behaviour flags.